src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.scss

```diff
@@ -120,3 +129,15 @@ md-icon {
   color: $delicate;
   font-size: $subhead-font-size-base;
 }
+
+.kd-sparkline {
+  width: 100px;
+  height: 20px;
+  vertical-align: -5px;
```

Comment:
It turns out to look fine without the baseline (this was a vestige of an earlier layout strategy) - removed in becde9ed735aa9274c1578b0ec70f8a2f9acc812